packages/authx/src/graphql/mutation/createClients.ts

```diff
@@ -43,12 +45,20 @@ export const createClients: GraphQLFieldConfig<
     }
   },
   async resolve(source, args, context): Promise<Promise<Client>[]> {
-    const { pool, authorization: a, realm } = context;
+    const { executor, authorization: a, realm } = context;
 
     if (!a) {
       throw new ForbiddenError("You must be authenticated to create a client.");
     }
 
+    const strategies = executor.strategies;
+    const pool = executor.connection;
+    if (!(pool instanceof Pool)) {
```

Comment:
Since we do this a lot, we could change this to be some kind of `executor.pool` getter.